tests/link/jax/test_basic.py

```diff
@@ -189,7 +189,8 @@ def test_jax_ifelse():
     compare_jax_and_py(x_fg, [])
 
     a = dscalar("a")
-    a.tag.test_value = np.array(0.2, dtype=config.floatX)
+    with pytest.warns(FutureWarning):
+        a.tag.test_value = np.array(0.2, dtype=config.floatX)
```

Comment:
These tests jax/numba don't need the tag stuff. I have no idea why they were written like this tbh

Reply:
So are you suggesting deleting the line ```a.tag.test_value = np.array(0.2, dtype=config.floatX)``` from the test itself?

Reply:
Yes, we are adding the value to the tag and then retrieving it when we call the function below. We can just pass it directly

Reply:
What about cases like this?
```
with pytest.warns(FutureWarning):
        y = vector("y")
        y.tag.test_value = np.r_[1.0, 2.0].astype(config.floatX)
        x = vector("x")
        x.tag.test_value = np.r_[3.0, 4.0].astype(config.floatX)
        A = matrix("A")
        A.tag.test_value = np.empty((2, 2), dtype=config.floatX)
        alpha = scalar("alpha")
        alpha.tag.test_value = np.array(3.0, dtype=config.floatX)
        beta = scalar("beta")
        beta.tag.test_value = np.array(5.0, dtype=config.floatX)

    # This should be converted into a `Gemv` `Op` when the non-JAX compatible
    # optimizations are turned on; however, when using JAX mode, it should
    # leave the expression alone.
    out = y.dot(alpha * A).dot(x) + beta * y
    fgraph = FunctionGraph([y, x, A, alpha, beta], [out])
    compare_jax_and_py(fgraph, [get_test_value(i) for i in fgraph.inputs])
    
   ```
   
Here, setting the ```test_value``` and then extracting the value from it actually comes handy when there are mutiple inputs to the function.

However we can also write this as:
```
with pytest.warns(FutureWarning):
        y = vector("y")
        x = vector("x")
        A = matrix("A")
        alpha = scalar("alpha")
        beta = scalar("beta")
        test_values = [np.r_[1.0, 2.0].astype(config.floatX), np.r_[3.0, 4.0].astype(config.floatX), np.empty((2, 2), dtype=config.floatX), np.array(3.0, dtype=config.floatX), np.array(5.0, dtype=config.floatX)]

    # This should be converted into a `Gemv` `Op` when the non-JAX compatible
    # optimizations are turned on; however, when using JAX mode, it should
    # leave the expression alone.
    out = y.dot(alpha * A).dot(x) + beta * y
    fgraph = FunctionGraph([y, x, A, alpha, beta], [out])
    compare_jax_and_py(fgraph, [i for i in test_values])
```

which method should i go aahead with? @ricardoV94

Reply:
The one without test values is perfectly fine. If you want you can have a dictionary from variables to values, if that seems more readable